SimMuon/GEMDigitizer/python/muonGEMDigis_cfi.py

```diff
@@ -7,10 +7,10 @@
 premix_stage2.toModify(simMuonGEMDigis, mixLabel = "mixData")
 
 from Configuration.Eras.Modifier_run2_common_cff import run2_common
-run2_common.toModify( simMuonGEMDigis, instLumi = 1.5)
+run2_common.toModify( simMuonGEMDigis, instLumi = 1.5, bx0filter = True)
```

Comment:
Should you keep it to False, so Run-2 behaviour will not be changed?

Reply:
There were only 5 GE1/1 super chambers for the slice test.
And as far as I know, they didn't support the multi bx mode at that moment.
@jshlee Could you confirm it?

Reply:
bx0filter = True is correct for run2

Reply:
OK, from https://github.com/cms-sw/cmssw/pull/34840#issuecomment-899366332 you expect to change the behaviour of Run-2.